### PR TITLE
Update language in NoSql persistence introduction

### PR DESCRIPTION
**Pull request by seangwright**
I was reading through this doc after going to a CosmosDb talk at a meetup.

I wanted to make some grammar, phrasing, and word choice updates/suggestions.

## Summary

Changed

*There are always constraints in the different databases will push you back*

to

*There are always constraints and trade-offs in the different databases technologies*

and

*querying multiple child collection properties is awful*

to

*querying multiple child collection properties is not easily optimized*

### CODE REVIEW

**Comment by mairaw** [comment]
I like the changes that you've made @seangwright! Thanks! @mvelosop @CESARDELATORRE given this is the e-book, do you want to review this change?

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/nosql-database-persistence-infrastructure.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ However, when you use a NoSQL database, especially a document-oriented database
...
 
-The difference is in how you persist that model. If you implemented your domain model based on POCO entity classes, agnostic to the infrastructure persistence, it might look like you could move to a different persistence infrastructure, even from relational to NoSQL. However, that should not be your goal. There are always constraints in the different databases will push you back, so you will not be able to have the same model for relational or NoSQL databases. Changing persistence models would not be trivial, because transactions and persistence operations will be very different.
+The difference is in how you persist that model. If you implemented your domain model based on POCO entity classes, agnostic to the infrastructure persistence, it might look like you could move to a different persistence infrastructure, even from relational to NoSQL. However, that should not be your goal. There are always constraints and trade-offs in the different databases technologies, so you will not be able to have the same model for relational or NoSQL databases. Changing persistence models is not a trivial task, because transactions and persistence operations will be very different.
```
Comment (rpetrusha) [inline docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/nosql-database-persistence-infrastructure.md]: databases technologies --> database technologies

**Review by rpetrusha — CHANGES_REQUESTED** [review]
I'm sorry that this has taken so long, @seangwright. There's one small change to make, and then this will be ready to merge.

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/nosql-database-persistence-infrastructure.md (hunk 1 above)
Comment (seangwright) [reply docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/nosql-database-persistence-infrastructure.md]: No problem. Fixed that typo.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional change, @seangwright. I'll merge your PR now. Your changes should be live on docs.microsoft.com in the next day or two.

**Pull request merged**